config/initializers/safe_mode_jail.rb

```diff
@@ -1,3 +1,7 @@
 class URI::Generic::Jail < Safemode::Jail
   allow :host, :path, :port, :query, :scheme
 end
+
+class Hash::Jail < Safemode::Jail
```

Comment:
I wonder why do we have these among initializers? Should those extensions be somewhere in `lib/foreman/renderer` folder? And if so, would it make sense to document allowed methods here [1] as well?

[1] - https://github.com/theforeman/foreman/blob/develop/lib/foreman/renderer/doc_templates/basic_ruby_methods.rb